test/e2e_node/container_log_rotation_test.go

```diff
@@ -105,4 +85,72 @@ var _ = SIGDescribe("ContainerLogRotation", framework.WithSlow(), framework.With
 			}, rotationConsistentlyTimeout, rotationPollInterval).Should(gomega.BeNumerically("<=", testContainerLogMaxFiles), "should never exceed max file limit")
 		})
 	})
+
+	ginkgo.Context("when a container log rotation with custom directory path", func() {
+		logDir, err := os.MkdirTemp("", "log-rotate-custom-path")
```

Comment:
What happens if you don't create a temp directory? I think kubelet creates the directory?

Reply:
Kubelet will use default path - `/var/log/pods/`.
Here we intentionally want a custom one to verify that logs land there.